common/app/routes/Challenges/Help-Modal.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Modal } from 'react-bootstrap';
+
+import ns from './ns.json';
+import {
+  createQuestion,
+  openHelpChatRoom,
+  closeHelpModal,
+  helpModalSelector
+} from './redux';
+
+const mapStateToProps = state => ({ isOpen: helpModalSelector(state) });
+const mapDispatchToProps = { createQuestion, openHelpChatRoom, closeHelpModal };
+const methodologyUrl = 'https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307'; // eslint-disable-line max-len
+
+const propTypes = {
+  closeHelpModal: PropTypes.func,
+  createQuestion: PropTypes.func,
+  isOpen: PropTypes.bool,
+  openHelpChatRoom: PropTypes.func
+};
+
+export class HelpModal extends PureComponent {
+  render() {
+    const {
+      isOpen,
+      closeHelpModal,
+      openHelpChatRoom,
+      createQuestion
+    } = this.props;
+    return (
+      <Modal
+        show={ isOpen }
+        >
+        <Modal.Header className={ `${ns}-list-header` }>
```

Comment:
This header has a white background. It should reflect the freeCodeCamp branding and be `@brand-primary` (less) `#006400` (hex) with `color: #fff;`

Reply:
Hope this looks fine: https://i.imgur.com/U9ES2GM.png

There appears to be a bug with namespaces. Several references in CSS to `.challenge-` where it should have been `.challenges-`. I only changed the rules related to modals. I'm not sure what effect will the following rules have so I'm leaving them inactive.

```diff
modified   client/less/code-mirror.less
@@ -27,7 +27,7 @@
   padding-bottom: 30px;
 }
 
-.challenge-log .CodeMirror {
+.challenges-log .CodeMirror {
   height: 100%;
   width: 100%;
 }
modified   client/less/main.less
@@ -844,7 +844,7 @@ code {
       color: darken(@brand-primary, 5%);
     }
   }
-  a, .input-group-addon, .challenge-instructions a, .challenge-instructions #MDN-links a {
+  a, .input-group-addon, .challenges-instructions a, .challenges-instructions #MDN-links a {
     color: #f8f8f8;
   }
   .fa-github-square {
```

Reply:
This is out of scope of this PR since this is an issue with the arrangement of these files. In a separate PR we should co-locate client/less/code-mirror.less in the Challenges directory since this file is directly related to this feature only.

Reply:
To be clear: the changes highlighted in the previous comment are not included in the PR. I'm just pointing out a potential bug for future reference. Sorry for the confusion.